.github/workflows/build.yaml

```diff
@@ -87,6 +91,17 @@ jobs:
           if [[ ${{ steps.changes.outputs.changes }} == true ]]; then
             ./scripts/test.sh ${{ matrix.crate }}
           fi
+
+  check-typos:
+    name: typos
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
What does this do? Does it throw error if it finds typos?

Reply:
yup , built with rust https://crates.io/crates/typos-cli

c.f. https://github.com/taiki-e/install-action

Reply:
it is configurable if there are false positives

Reply:
Alright, I guess we can have it.